src/routes/formsg/formsgSiteLaunch.ts

```diff
@@ -150,6 +151,7 @@ export class FormsgSiteLaunchRouter {
     this.usersService = usersService
     this.infraService = infraService
     // We need to bind all methods because we don't invoke them from the class directly
+    nameAnonymousMethods(this)
     autoBind(this)
```

Comment:
Note on this for reviewer: `autoBind(this)`in this file (and all the router files thathave been converted to typescript) doesn't actually do what we think it does 😑😢 

When the methods are declared like this in the class:
```
 handleSiteLaunchFormRequest: RequestHandler<
    never,
    string,
    { data: { submissionId: string } },
    never,
    { submission: DecryptedContentAndAttachments }
  > = async (req, res) => {
   //. ...
}
```
then they are:
1) anonymous
2) **NOT added to the class' prototype**, so `autoBind()` will not get to them at all